app/assets/stylesheets/_colours.scss

```diff
@@ -0,0 +1,78 @@
+$black:         #000;
```

Comment:
Good idea, will make that a separate Issues & PR, I think. Home page is probably blocked for a little while, and good to get the SCSS in anyway.